docs/guides/operator-guide-azure-enclave.md

```diff
@@ -104,23 +98,22 @@ To deploy a new UID2 Private Operator for Azure, you'll need to complete the fol
 
 ### Download ZIP File and Extract Files
 
-The first step is to get set up with the deployment files you'll need.
+The first step is to get set up with the deployment files you'll need. Download the ZIP file linked in the following table, Azure Download column, for the latest version. 
```

Comment:
Can I suggest:
Download the ZIP file linked in the following table in the Azure Download column for the latest version.